apps/web/src/lib/tiptap/extensions/trailing-node.ts

```diff
@@ -0,0 +1,63 @@
+import { Extension } from "@tiptap/core";
+import { Node, NodeType } from "@tiptap/pm/model";
+import { Plugin, PluginKey } from "prosemirror-state";
+
+// based on https://github.com/ueberdosis/tiptap/blob/40a9404c94c7fef7900610c195536384781ae101/demos/src/Experiments/TrailingNode/Vue/trailing-node.ts
```

Comment:
Should these comments stay?

Reply:
I think it can be nice for future reference!